README.md

```diff
@@ -27,7 +27,7 @@ Download Papercut.Setup.exe installer.
 
 ## Papercut SMTP Background Service
 Papercut SMTP has an optional "always on" service to receive emails even when the client is not running. It's installed by default with [Papercut.Setup.exe](https://github.com/ChangemakerStudios/Papercut/releases).
-Alternatively, it can be run in an almost portable way by downloading [Papercut.Service.zip](https://github.com/ChangemakerStudios/Papercut/releases), unzipping and [following the service installation instructions](https://github.com/ChangemakerStudios/Papercut/tree/develop/src/Papercut.Service).
+<br />Alternatively, it can be run in [an almost portable way](https://github.com/ChangemakerStudios/Papercut-SMTP/issues/248) by downloading [Papercut.Service.zip](https://github.com/ChangemakerStudios/Papercut/releases), unzipping and [following the service installation instructions](https://github.com/ChangemakerStudios/Papercut/tree/develop/src/Papercut.Service#readme).
```

Comment:
Markdown supports two linebreak syntaxes, meaning we don't have to fall back to HTML here.

`\` at the end of the previous line would ensure a line break within the paragraph.

Reply:
[Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) doesn't it, but you're right, it works. Yet why closing the entire PR because of it?

Anyway, I've fixed it in #270.

Reply:
If you want this merged, you need to be merged with the latest revisions to the README.

Reply:
Done, moved to #271.